server/controllers/auth.js

```diff
@@ -32,6 +33,22 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
     const token = generateToken(user._id);
     const secondsInWeek = 604800;
 
+    // create default lists
+    await List.create({
```

Comment:
Because nothing else in this function depends on these two lists, you should be fine creating them without awaiting